doc/classes/Bone2D.xml

```diff
@@ -34,7 +34,7 @@
 		<method name="get_default_length" qualifiers="const">
 			<return type="float" />
 			<description>
-				Deprecated. Please use  [code]get_length[/code] instead.
+				Deprecated. Please use [method get_length] instead.
```

Comment:
👀

We can probably remove these and avoid release 4.0 with already deprecated methods with a clear replacement :)

Reply:
It was deprecated in... https://github.com/godotengine/godot/pull/47872. That's probably enough time. I think it was originally a property but now only the methods remain, and they are completely redundant. We can add them to the convertor, which we probably didn't have before.

Reply:
PR'd https://github.com/godotengine/godot/pull/73513